src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -45,12 +55,38 @@ public void playGame(){
             }
         }
 
+	if(board.getWinner() == Player.X){
+            score[0] += 1;
+	}
+	else if (board.getWinner() == Player.O){
+            score[1] += 1;
+	}
+	else{
+            score[2] += 1;
+	}
         board.printBoard();
         System.out.println("Player " + board.getWinner() + " has won the game!");
+        printScore();
+    }
+
+    public boolean promptNewGame() {
+        Scanner keyboardScanner = new Scanner(System.in);
+        System.out.println("Do you want to play another game (y/n)?");
+        String line = keyboardScanner.nextLine().toLowerCase();
+        if (line.contentEquals("y")) {
+            return true;
+        } else {
```

Comment:
I think you should add a User Input Error Handling.
If a user wants to input "y", but user inputs some other character by mistake. In this case, I think you should ask the user again.